modal-go/function.go

```diff
@@ -85,6 +86,7 @@ func (function *Function) Remote(ctx context.Context, args []any, kwargs map[str
 		Input: pb.FunctionInput_builder{
 			Args:       payload.Bytes(),
 			DataFormat: pb.DataFormat_DATA_FORMAT_PICKLE,
+			MethodName: &function.MethodName,
```

Comment:
Since this is an optional field, it would be great if we could represent that in the `Function` as well, by having MethodName be a `*string` (just so we can construct the exact same protobuf that the client sends)